_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What's the point of a simple theory?
+#subtitle: A story about the power of null models to shape how we interpret data
+#status: inactive #active if you want it on the frontpage
+
+category: paper
+
+description: |
+  A story about the power of null models to shape how we interpret data.
+
+people: # add people that are involved in this project
+  - maddy
+
+layout: blog # do not change this
+image: '/img/posts/why-null-models-crispr/banfield_data_theory.svg'
+#link:
+no-link: false # if you don't want it's own webpage
+last-updated: 2022-1-12
+# do NOT add 0 (such as 06 for June). Will break everything lol
+# take out the underscore in the name.
+---
+
+Experimentalists wade through the murky waters of biology, hoping to tease out knowledge by controlling the things they can control and carefully observing what they cannot. In a well-designed experiment, the murky water is illuminated and an insight is reached by painstakingly ruling out other explanations. This might sound logical and systematic, but how do you know when you're done considering reasonable alternatives? And in biology, where some [truly unreasonable](https://www.theatlantic.com/science/archive/2017/06/jumping-spiders-can-see-the-moon/529329/) things can happen, how do you know what's 'reasonable' anyway?
+
+When there are many possible explanations for an experimental observation, theory can help. There are lots of ways to design a theoretical model of a biological system, and how you do it depends a lot on what you want to find out. On one extreme, very *mechanistic* models try to include every known biological detail to the highest accuracy possible - these models are often trying to calculate specific numbers (such as [how many COVID-19 cases your region will see in the next few weeks](https://covid19-sciencetable.ca/sciencebrief/update-on-covid-19-projections-15/)). The type of model we'll discuss here falls on the other extreme: *phenomenological* models whose main concern is capturing an interesting *phenomenon*, removing as much biological 'clutter' as possible to be sure that you really understand *why* things happen in your model. These models are [definitely wrong](https://en.wikipedia.org/wiki/All_models_are_wrong), but we hope they are wrong in a way that helps us understand or rule out some of the possible explanations for experimental observations. 
```

Comment:
I think I know what you mean when you say "they are wrong in a way that helps us understand ....", but maybe it's more accurate to say they hold some grain of truth? It's not the fact that it's wrong that lets us understand what's going on (your choice to keep it in, I think the way you wrote it makes for a very nice picture still!)

Reply:
Haha good point, but I would say that it's the deliberate ignoring of certain biological "truth" in these models (their wrongness) that makes them simple enough to understand!

Reply:
I agree! Keep it as such, I think it's good phrasing to plant this little thought in peoples mind.